themes/default/content/what-is/what-is-infrastructure-as-code.md

```diff
@@ -62,85 +62,105 @@ video:
 
-Infrastructure as code is the latest step in the evolving process of defining and managing infrastructure. Before infrastructure as code, infrastructure was (and in some cases still is!) provisioned by many methods such as pointing and clicking in a user interface, batch scripts, and configuration management tools that may not have been designed by the cloud. Today, modern approaches use platforms, such as [Pulumi](/), which embrace and support the full software engineering lifecycle.
+Before infrastructure as code, infrastructure was (and in some cases still is!) provisioned in a variety of ways, such as by pointing and clicking in a user interface (UI), by running commands via a command-line interface (CLI), by running batch scripts, or by using configuration management tools that may not have been designed with cloud infrastructure in mind. Each of these methods falls short in some way; interactive methods involving a UI or a CLI often create problems with repeatability and consistency while batch scripts or configuration management tools may be unable to declaratively manage infrastructure. Today, modern approaches use platforms, such as [Pulumi](/), which embrace and support the full software engineering lifecycle.
 
-The term “infrastructure as code” or IaC has become commonplace, but there are important questions attached to it. What is infrastructure as code really? What are the benefits of infrastructure as code?  How do you integrate infrastructure as code into your organization? In this article, we’re going to touch on all these points and discuss why infrastructure as code matters for modern application development.
+In this article, we'll touch on three key aspects regarding infrastructure as code:
 
-## Why Has Infrastructure as Code Become Important?
+* Why is infrastructure as code important?
+* What are the key elements of infrastructure as code?
+* What benefits does infrastructure as code provide?
+* How do you get started with infrastructure as code?
 
-Infrastructure as code matters because of three significant trends, all of them happening at the same time.
+Let's start with examing why infrastructure as code is important.
 
-### The transition to the cloud
+## Why is Infrastructure as Code Important?
+
+Infrastructure as code is important because of three significant trends, all of them happening at the same time.
+
+### The Transition to the Cloud
 
 One trend, of course, is the ongoing transition to the cloud. More and more companies are shifting workloads from on-premises infrastructure to cloud environments. Cloud-based infrastructure is provisioned via APIs, and, as a result, can be easily managed with infrastructure as code tools.
 
-### Cloud modernization
+### Cloud Modernization
```

Comment:
is transition to cloud and cloud modernization sorta redundant with each other? i wonder if the other trend should be more process or people oriented

Reply:
For the short-term future, I'm of the opinion that "transitioning to the cloud" and "cloud modernization" (or perhaps we could use "cloud optimization") are different and distinct. The first captures those companies that are largely involved in a "lift-and-shift" effort to get onto a cloud provider, whereas the second involves a greater level of cloud maturity and natively leveraging cloud services in a more effective and efficient way. Does that make sense?